### PR TITLE
DYN-4135-UnpinFromNode Added to ContextMenu

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

I added the new option "unpin from node" in the Note Context Menu and also added the needed resources.
Also I added a new method CanUnpinFromNode so it will manage when the option will be enabled and disabled in the context menu.
Finally I did a fix in NoteView.xaml.cs due that every time that the right click was pressed over the note a new DoubleInput (number) node was added to the selection so it was making the CanUnpinFromNode() method to be non working in some cases.

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

 Added the new option "unpin from node" in the note Context Menu

### Reviewers

@QilongTang 

### FYIs

@filipeotero

### CODE REVIEW

**Comment by RobertGlobant20** [comment]
GIF of how the new option looks in the context menu.
unpin_note

File: src/DynamoCoreWpf/Views/Core/NoteView.xaml.cs [hunk 1]
```diff
@@ -98,7 +98,8 @@ void OnNoteTextPreviewMouseDown(object sender, MouseButtonEventArgs e)
            if (this.ViewModel.Model.PinnedNode != null)
             {
             var nodeGuid = this.ViewModel.Model.PinnedNode.GUID;
-                DynamoSelection.Instance.Selection.Add(ViewModel.Model.PinnedNode);
+                //We have to use AddUnique due that otherwise the node will be added several times when we click right over the note
+                DynamoSelection.Instance.Selection.AddUnique(ViewModel.Model.PinnedNode);
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Core/NoteView.xaml.cs:102]: 👍🏻

File: src/DynamoCoreWpf/Views/Core/NoteView.xaml [hunk 2]
```diff
@@ -76,6 +76,14 @@
                         <ToolTip Content="Select a node to pin to this note" Style="{StaticResource GenericToolTipLight}"/>
                     </MenuItem.ToolTip>
                 </MenuItem>
+                <MenuItem Name="UnpinToNode"
+                          Header="{x:Static p:Resources.ContextUnpinFromNode}" 
+                          Command="{Binding Path=UnpinFromNodeCommand}"
+                          ToolTipService.ShowOnDisabled="True">
+                    <MenuItem.ToolTip>
+                        <ToolTip Content="Unpin this note from the node" Style="{StaticResource GenericToolTipLight}"/>
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Core/NoteView.xaml:84]: Tooltip content need to be added as resource string as well for localization

**Review by QilongTang — COMMENTED** [review]
One comment then LGTM

File: src/DynamoCoreWpf/Views/Core/NoteView.xaml (hunk 2 above)
Comment (RobertGlobant20) [reply src/DynamoCoreWpf/Views/Core/NoteView.xaml]: @QilongTang tooltip was added as resource in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12578/commits/e87e3c0711eddfeeb5c8c780be72c352ed5e22fe

**Review by QilongTang — APPROVED** [review]

**Pull request merged**